### ISSUE TITLE
What accessibility standards should be the requirement?

### ISSUE THREAD

**Issue by krilor** [krilor/wordpress-website-standards#1]
I'm a proponent of using one of the standards available in [AChecker](https://achecker.ca/checker/index.php) to make it testable?

WordPress core seems to be using WCAG 2.0 (AA). Ref https://make.wordpress.org/accessibility/handbook/test-for-web-accessibility/

* BITV 1.0 (Level 2)
* Section 508
* Stanca Act
* WCAG 1.0 (Level A)
* WCAG 1.0 (Level AA)
* WCAG 1.0 (Level AAA)
* WCAG 2.0 (Level A)
* WCAG 2.0 (Level AA)
* WCAG 2.0 (Level AAA)
